uaclient/util.py

```diff
@@ -669,3 +670,13 @@ def check_lock_info(lock_path: str) -> "Tuple[int, str]":
         )
         os.unlink(lock_path)
         return no_lock
+
+
+def copy_file(src: str, dst: str) -> None:
```

Comment:
I suppose we don't really need this function if you end up using cfg.read_cache -> write_cache calls. I'll let you digest the comments above and determine if it's still worth it.

Reply:
Yes, you are right. The combination of those cache functions is already enough